classes/HydraCore.php

```diff
@@ -105,20 +112,14 @@ public static function awesomeIcon( $name, array $extraClasses = [], array $extr
 	/**
 	 * Returns the number of users who have made at least one edit on the wiki.
 	 */
-	public static function numberofcontributors() {
-		$cache = MediaWikiServices::getInstance()->getMainWANObjectCache();
-		$key = $cache->makeKey( 'NumberOfContributors' );
-		$hit = $cache->get( $key );
+	public function numberOfContributors() {
+		$key = $this->cache->makeKey( 'NumberOfContributors' );
+		$hit = $this->cache->get( $key );
 		if ( !$hit ) {
-			if ( class_exists( \ActorMigration::class ) ) {
-				$actorQuery = \ActorMigration::newMigration()->getJoin( 'rev_user' );
-				$userField = $actorQuery['fields']['rev_user'];
-			} else {
-				$actorQuery = [ 'tables' => [], 'joins' => [] ];
-				$userField = 'rev_user';
-			}
+			$actorQuery = [ 'tables' => [], 'joins' => [] ];
+			$userField = 'rev_user';
```

Comment:
As per https://www.mediawiki.org/wiki/Manual:Revision_table#rev_user
or https://www.mediawiki.org/w/index.php?title=Manual:Database_layout/diagram&action=render
This column has been removed